spec/support/unit/helpers/active_record_versions.rb

```diff
@@ -11,42 +11,6 @@ def active_record_version
       Tests::Version.new(::ActiveRecord::VERSION::STRING)
     end
 
-    def active_record_enum_supports_prefix_and_suffix?
```

Comment:
@VSPPedro Is this what you are referring to when you commented about legacy controller callbacks?

I think removing all the legacy callbacks under rails < 5 in this PR would be good.

Reply:
I'm so sorry. I thought you hadn't made that change. I should have analyzed the changes better.

> I think removing all the legacy callbacks under rails < 5 in this PR would be good.

I agree.